runtests.py

```diff
@@ -274,16 +274,17 @@ def add_stdlibsamples(driver: Driver) -> None:
                 modules.append(module)
         if modules:
             driver.add_mypy_modules('stdlibsamples (%s)' % (version,), modules,
-                                    cwd=stdlibsamples_dir)
+                                    cwd=stdlibsamples_dir, extra_args=['--no-strict-optional'])
 
 
 def add_samples(driver: Driver) -> None:
     for f in find_files(os.path.join('test-data', 'samples'), suffix='.py'):
         if f == os.path.join('test-data', 'samples', 'crawl2.py'):
             # This test requires 3.5 for async functions
-            driver.add_mypy_cmd('file {}'.format(f), ['--python-version=3.5', f])
+            driver.add_mypy_cmd('file {}'.format(f), ['--python-version=3.5',
```

Comment:
I am not sure we actually need the `--python-version=3.5` flag. I thought mypy now uses the latest Python version by default.